Create virtual-onboarding.md (#1806)
alan-turing-institute/the-turing-way: book/website/collaboration/new-community/new-community-onboarding.md

(cl-new-community-onboarding)=
# Virtual Onboarding of New Members

## Summary

Joining a new group virtually is a very different experience from joining in person, and it can be hard to know if someone is struggling or isolated.  
You can help to make this experience less overwhelming by having a clear onboarding plan and resources. 

## Introducing the Team

> It can be intimidating to be the first to speak as a new member - consider asking your team to do a "who's who", and to introduce themselves first in meetings.

Author: I have removed this text - thank you Emma

Names and roles are also easy to forget, particularly if we aren't seeing people in the office every day. 

> It can be helpful to store this information somewhere, for example, by sending around an introductory email.

Reviewer: ```suggestion
It can be helpful to store this information somewhere, for example, by sending around an introductory or onboarding email.
```

Also consider organising online social events, such as coffee chats or team activity like a video game or escape room to help new members integrate into the group. 

An onboarding event sharing important information like contact details, organigram, shared calendar, resources available to everyone, organisation policy and social norms can be extremely useful.

## Support

You can provide one-on-one support - such as a buddy for informal peer support, or a mentor, for more formal support.

> This helps the new joiner understand group norms, and allows them to ask any sensitive questions they wouldn't like to raise in a group discussion.

Reviewer: ```suggestion
This helps the new joiner understand group norms, and allows them to ask any sensitive questions they wouldn't like to raise in a group discussion.
You should consider that the buddy or mentor should remain for a longer period than would be needed with in person onboarding.

Co-working with a buddy or several team members is another informal way to build relationships with new team members and is an informal environment to address any questions that may have arisen in the first few days or weeks of employment. 
```

## Onboarding Document Guidelines

> Your project can have an onboarding document or manual. 

Reviewer: ```suggestion
Your project can have an onboarding document, manual or folder of resources. 
```

> This creates a set of shared expectations and acts as a reference for new members to find information about resources, or who to ask for help if they have a problem.

Reviewer: ```suggestion
This creates a set of shared expectations and acts as a reference for new members to find information about resources. It should also contain information about who to ask for help if they have a problem.
```

### Suggested Sections and Content

#### Group Culture

Every group or workplace has a different culture, and expectations aren't always clear when a new member joins. 

> You can help them settle in by making the culture clear here. 

Reviewer: ```suggestion
You can help them settle in by making the culture clear in written form. 
```

This can be especially important for people who experience discrimination - do they know that they can approach you and be listened to if they experience harassment or discrimination?

Group culture sections can include:

> * What members of the group can expect from you - turnaround times for feedback, support.

Reviewer: Who do you mean by you? Leadership group? The project lead? Line managers? I think this needs to be made more clear

> * What you expect from members of the group - timekeeping, communication. This may vary between different roles.

Reviewer: Maybe change this to what is expected as part of this team in general (for everyone to do) and then another section of what is expected in certain team roles.

Author: I have merged these two together and positioned it from the lens of the team in general - thank you!

> * Expectations for working hours.

Reviewer: I think it is important here to mention flexible working and the fact that this needs to be made explicit from the start so that it is fair for all in the team/workplace.

Author: Absolutely - thanks for pointing this out. I've included some mention of this and we will also be covering this more when restructuring the chapter and sub chapters!

> * Equality, inclusion and diversity statement.

Reviewer: Mention code of conduct in general here

Author: Done!

* Open science and how you expect the group to practice this.

#### Key Contacts

It can be hard to ask for help, particularly when working remotely - you can't just lean over to the person at the next desk and ask! 

Key contacts could include:

> * Group technicians/managers

Reviewer: ```suggestion
* Group technicians/managers.
```

* Key collaborators, and info on what areas of the project they are involved in.
* IT - include detail - is there a form, or a specific person to email?

#### Communication Channels

- Preferred method of communications, _for example email, video call, slack message_

> - Ensure any communication preferences are ..communicated.. from the beginning

Reviewer: ```suggestion
- Ensure any communication preferences are ..communicated.. from the beginning. 
It should be a priority that the new team member is sent all relevant information to join these communications channels and someone should check that they have been able to set them up. 
Offer help with this process and also training on any communications channel that the new team member has not used before so that they can communicate with the other team members quickly after joining. 
```

#### Accessing Shared Documents

- What method of sharing is preferred? _for example google docs, RMarkdown, shared documents over email, GitHub, OneDrive_ 

> - Ensure invitations are set up for calendar meetings and access is in place

Reviewer: ```suggestion
- Ensure invitations are set up for calendar meetings and access is in place.
```

#### Expectations in Meetings

> - Individuals may not have adequate access to equipment/internet. 

Reviewer: ```suggestion
- Individuals may not have adequate access to equipment or the internet. 
```

Discussions about whether any support is required for home setup (for example webcams, microphone, laptop) can allow for any needs to be addressed.
- Meeting etiquette - learning expectations for different meetings can be overwhelming for a new starter. 

> Clearly outlining expected attendance and contribution can help overcome this, _for example are attendees required to read the papers prior to journal club? Are attendees asked to present their work regularly? Are you expecting videos to be on?_

Reviewer: ```suggestion
Clearly outlining expected attendance and contributions can help overcome this, _for example, are attendees required to read the papers prior to journal club? Are attendees asked to present their work regularly? Are you expecting videos to be on?_
```

- Meeting times - consider the circumstances of your group when organising regular meetings - does everyone have the same opportunity to attend? _for example some groups try to arrange Athena SWAN meeting times, whereby meetings occur between 10-3, meaning that those with children are more able to attend this_.

#### Training

> On tools, the group uses often - for example GitHub

Reviewer: ```suggestion
On tools, the group uses often - for example GitHub and communication channels such as Slack.
```

> * Use Zenodo or similar to host training materials for easier sharing 

Reviewer: ```suggestion
* Use [Zenodo](https://zenodo.org/), or a similar open repository, to host training materials for easier sharing. 
```

> * Use a Youtube channel to store induction & training for people to watch in their own time

Reviewer: ```suggestion
* Use a Youtube channel to store induction & training material for people to watch in their own time.
```

> * Consider running training & induction sessions every 2-3 months as new people join

Reviewer: ```suggestion
* Consider running training & induction sessions every 2-3 months as new people join.
```

> * Consider a Slack channel for new joiners - for questions and discussion 

Reviewer: ```suggestion
* Consider a Slack channel for new joiners - for questions and discussion. 
```

> * Consider providing Office Hours to support new joiners using new tools - for example, GitHub

Reviewer: ```suggestion
* Consider providing Office Hours to support new joiners using new tools - for example, GitHub.
```

> * Get feedback on your training
> * Try and make your training session as interactive as possible with exercises 

Reviewer: ```suggestion
* Try to make your training sessions as interactive as possible with exercises and discussions.
* Get feedback on your training and use this to adjust and improve your resources.
```

#### Resources 

> * Link to the group website/Twitter
> * Links to other resources - institution wellbeing services.

Reviewer: ```suggestion
* Link to the group website.
* Link to group social media accounts - Twitter, Facebook, Instagram.
* Links to other resources - institution wellbeing services.
* Links to group policies.
```
